src/sphinx/extensions/gradle_plugin.rst

```diff
@@ -0,0 +1,358 @@
+|                       |         |    { include "**/*Simulation*.scala" }                                                                                                                                                                                                                                                      |                                                                                                 |
++-----------------------+---------+---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+-------------------------------------------------------------------------------------------------+
+
+How to override Gatling version, JVM arguments and system properties
+
+.. code-block:: groovy
+
+   gatling {
+     toolVersion = '3.4.0'
+     jvmArgs = ['-server', '-Xms512M', '-Xmx512M']
+     systemProperties = ['file.encoding': 'UTF-8']
+   }
+
+How to filter simulations
+
+.. code-block:: groovy
+
+   gatling {
+     simulations = {
+       include "**/package1/*Simu.scala"    // <1>
+       include "**/package2/*Simulation.scala"  // <2>
+     }
+   }
+
+1. all Scala files from plugin simulation dir subfolder ``package1`` ending
+   with ``Simu``.
+2. all Scala files from plugin simulation dir subfolder ``package2`` ending
+   with ``Simulation``.
+
+Gatling configuration
+=====================
+
+Override gatling.conf settings
+******************************
+
+To override
+`default parameters <https://github.com/gatling/gatling/blob/master/gatling-core/src/main/resources/gatling-defaults.conf>`_
+of Gatling just put own version of ``gatling.conf`` into ``src/gatling/resources``.
+
+Logging management
+******************
+
+Gatling uses `Logback <http://logback.qos.ch/documentation.html>`_ to customize
+its output. To change logging behaviour, put your ``logback.xml`` into resources
+folder, ``src/gatling/resources``.
+
+If no custom ``logback.xml`` provided, by default plugin will implicitly use
+following configuration.
+
+Default ``logback.xml`` created by the plugin
+
+.. code-block:: xml
+
+   <?xml version="1.0" encoding="UTF-8"?>
+   <configuration>
+     <appender name="CONSOLE" class="ch.qos.logback.core.ConsoleAppender">
+       <encoder>
+         <pattern>%d{HH:mm:ss.SSS} [%thread] %-5level %logger{36} - %msg%n</pattern>
+         <immediateFlush>false</immediateFlush>
+       </encoder>
+     </appender>
+     <root level="${logLevel}"> <!--1-->
+       <appender-ref ref="CONSOLE"/>
+     </root>
+   </configuration>
+
+1. ``logLevel`` is configured via plugin extension, ``WARN`` by default.
```

Comment:
Missing mention of the effect of `logHttp`.

Reply:
Wanted: it is not in the default configuration.

But you're right, I added another example.